run.sh

```diff
@@ -157,6 +162,10 @@ function parse_arguments() {
                 RUN_COMMAND="deploy"
                 shift
                 ;;
+            --upgrade)
```

Comment:
As above, the name of misleading.
We are not upgrading the submariner.

Please, change it to something like: --subm-catalog-update